pkgs/top-level/all-packages.nix

```diff
@@ -4470,6 +4470,8 @@ with pkgs;
 
   phodav = callPackage ../tools/networking/phodav { };
 
+  phonenumber = callPackage ../development/libraries/libphonenumber { };
```

Comment:
Use `libphonenumber` instead, and put it in the section of `all-packages.nix` where other libraries are.

Reply:
I thought this was alphabetical?